manager/runner.go

```diff
@@ -604,21 +618,42 @@ func (r *Runner) Run() error {
 			// Record that there is at least one new render event
 			newRenderEvent = true
 
+			var label string
```

Comment:
nitpick: since this function is getting super long, what do you think about extracting the logic for getting this label into a small helper function? 


```
func getTelemetryLabel(event *RenderEvent) string {
	var label string

	// Record that at least one template would have been rendered.
	if event.WouldRender {
		label = "would"
	}

	// Record that at least one template was rendered.
	if event.DidRender {
		label = "rendered"
	}

	if event.ForQuiescence {
		label = "quiescence"
	}

	return label
}
```
so here to end up with:

```
label := getTelemetryLabel(event)
```

Reply:
Suggestion applied!